research/object_detection/eval.py

```diff
@@ -78,6 +78,27 @@
 flags.DEFINE_boolean('run_once', False, 'Option to only run a single pass of '
                      'evaluation. Overrides the `max_evals` parameter in the '
                      'provided config.')
+
+# Performance tuning specific to MKL
```

Comment:
Strictly speaking, are these flags needed? It seems to me that they just set environment variables. Can the user set these variables before running the eval script?

Reply:
@derekjchow thanks for your useful comments! yeah, those are environment variables and parallelism control parameters, very similar to https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/benchmark_cnn.py (line 178, 323),  we are trying to improve user's OOB experiences while they use MKL.